### PR TITLE
NF: duplicate rename

### PR DESCRIPTION
**Pull request by Arthur-Milchior**

### CODE REVIEW

**Review by david-allison-1 — APPROVED** [review]

**Review by david-allison-1 — CHANGES_REQUESTED** [review]
CI failed

**Comment by Arthur-Milchior** [comment]
Removed conflict. I'll see CI if it fails again. It compiles on my machine

File: AnkiDroid/src/main/res/values/17-model-manager.xml [hunk 1]
```diff
@@ -30,6 +30,7 @@
     <!--Browser-->
     <string name="model_browser_delete">Delete note type</string>
     <string name="model_browser_rename">Rename</string>
+    <string name="model_browser_options">Options</string>
```
Comment (mikehardy) [inline AnkiDroid/src/main/res/values/17-model-manager.xml:33]: I understand the comment needed for duplicate rename, I don't understand the new string added here?

**Review by mikehardy — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/main/res/values/17-model-manager.xml (hunk 1 above)
Comment (Arthur-Milchior) [reply AnkiDroid/src/main/res/values/17-model-manager.xml:33]: Bad conflict resolution I assume

**Review by mikehardy — APPROVED** [review]

**Review by david-allison-1 — APPROVED** [review]

**Pull request merged**